backend/src/test/java/org/cryptomator/hub/api/VaultResourceTest.java

```diff
@@ -805,10 +804,34 @@ public void testUnlockBlockedIfLicenseExceeded() throws SQLException {
 			}
 			//Assumptions.assumeTrue(EffectiveVaultAccess.countEffectiveVaultUsers() > 5);
```

Comment:
Remove or adjust this line. (assumption is breaking the hard limit, which is not true)

Reply:
Fixed in https://github.com/cryptomator/hub/pull/258/commits/09b712a82c40fe092a9f8f774f189a96b69d5759